internal/uvm/constants.go

```diff
@@ -16,4 +19,15 @@ const (
 	DefaultVPMemSizeBytes = 4 * 1024 * 1024 * 1024 // 4GB
 )
 
-var errNotSupported = fmt.Errorf("not supported")
+// Unsupported/wrong UVM configuration errors
+var (
+	errNotSupported             = errors.New("not supported")
+	errSCSICount                = errors.New("SCSI controller count must be 0 or 1") // Future extension here for up to 4
+	errVPMEMCount               = fmt.Errorf("VPMem device count cannot be greater than %d", MaxVPMEMCount)
+	errVPMEMSize                = errors.New("VPMemSizeBytes must be a multiple of 4096")
+	errZeroVPMemVHD             = errors.New("PreferredRootFSTypeVHD requires at least one VPMem device")
+	errKernelDirectBootBuild    = errors.New("KernelDirectBoot is not supported on builds older than 18286")
+	errLayerFoldersNumber       = errors.New("at least 2 LayerFolders must be supplied")
+	errEnableDiscardHintBuild   = errors.New("EnableColdDiscardHint is not supported on builds older than 18967")
+	errPhysicallyBackedDeferred = errors.New("EnableDeferredCommit is not supported on physically backed VMs")
+)
```

Comment:
I'm not sure how I feel about having these. If the goal is only to be consistent between LCOW and WCOW for error messages, I'm not really sure how valuable this is. If we use these in some other way or have plans to use them in some other way, then that's fine. Can you elaborate here?